src/EPR.CommonDataService.Core/Services/SubmissionsService.cs

```diff
@@ -1,9 +1,15 @@
+#nullable disable
```

Comment:
here too

Reply:
We have new injections of Logger and Config.
it's complaining about possible null value for config.

There are no alerts anywhere in the code